packages/react-components/react-toast/src/state/types.ts

```diff
@@ -9,9 +9,16 @@ export interface ToastOptions {
   position?: ToastPosition;
   content?: unknown;
   timeout?: number;
+  pauseOnWindowBlur?: boolean;
+  pauseOnHover?: boolean;
 }
 
-export interface Toast extends Required<Omit<ToastOptions, 'toasterId'>> {
+export interface DefaultToastOptions
+  extends Pick<ToastOptions, 'position' | 'timeout' | 'pauseOnWindowBlur' | 'pauseOnHover'> {}
+
+export interface ValidatedToastOptions extends Required<DefaultToastOptions> {}
```

Comment:
I would vice-versa and make them required by default and and use `Partial<ToastOptions>` in public interface